app/controllers/orders_controller.rb

```diff
@@ -1,50 +1,42 @@
 class OrdersController < ApplicationController
-  @title = ''
-
   def index
-    @title = 'All orders'
     @order = Order.all
   end
 
   def new
-    @title = 'New Order'
     @order = Order.new
   end
-  
+
   def show
-    @order = Order.find(params[:id])
-    @title = "Order N #{@order.id}" if @order
+    @order = Order.find(params[:id]) rescue not_found if @order.nil?
```

Comment:
вот что я имел в виду
```suggestion
    @order = Order.find_by_id(params[:id])
    not_found if @order.nil?
```